### DISCUSSION TITLE
Support for new Fibaro Roller Shutter 4 FGR224

### DISCUSSION THREAD

**Discussion by LudovicP** [zwave-js/zwave-js#6842] [Feature Requests]
### Description

The new Fibaro FGR224 modules have been released and are replacing the FGR223 in all online stores.

### Do you have a solution in mind?

If someone can add required configuration for those modules to be recognized in zwave js, it should be cool.

**Comment by AlCalzone** [comment]
Duplicate of #6811. There is an [open PR](https://github.com/zwave-js/node-zwave-js/pull/6812) which is waiting for some help to get it in a releasable state.

**Reply by LudovicP** [reply]
Sorry, I didn't find it with the hyphen :/
